xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -160,6 +158,17 @@
       </AssemblyInfo>
       <Docs>
         <summary>Allows specific implementations of this abstract class to be instantiated.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Out of the box, the valid implementations names, and their associated implementations, are the following : 
```

Comment:
"out of the box" is an awkward phrase, and it would be good to time the name to the hashName parameter. So I'd change this to "The .NET Framework includes the implementations and their associated `hashName` values:"